jetpack/package.json

```diff
@@ -1,4 +1,8 @@
 {
 	"name": "jetpack",
-	"main": "mod.js"
-}
\ No newline at end of file
+	"main": "mod.js",
+	"index": "0.1.1",
```

Comment:
Why is this called `index` and not  `version`?

Reply:
...oh, whoops. Mistake while copying/pasting stuff around.